File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/UpdateLabtestsRequiredRequest.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class UpdateLabtestsRequiredRequest
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("Label")]
        public string? Label { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("RequiredLabTestBatches")]
        public List<object>? RequiredLabTestBatches { get; set; }
    }
}

File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/UpdateLabtestsRequiredRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;import java.util.List;
import com.fasterxml.jackson.annotation.JsonProperty;public class UpdateLabtestsRequiredRequest {
    @JsonProperty("Label")
    public String label;
    @JsonProperty("RequiredLabTestBatches")
    public List<Object> requiredLabTestBatches;
}
